OpenEdXMobile/src/main/java/org/edx/mobile/module/prefs/LoginPrefs.java

```diff
@@ -106,6 +107,19 @@ public void saveDefaultPlaybackSpeed(float speed) {
         pref.put(PrefManager.Key.PLAYBACK_SPEED, speed);
     }
 
+    public void setVideoQuality(VideoQuality videoQuality) {
```

Comment:
Do remove the saved preferences on user logout.
```line:68-> pref.put(PrefManager.Key.VIDEO_QUALITY, -1);```

Reply:
One more thing. I forgot to remove the user's profile image on logout as well in LEARNER-8438. Can we remove it in the scope of this ticket?
``` line:69 -> pref.put(PrefManager.Key.PROFILE_IMAGE, null);```

Reply:
Used `line:68-> pref.put(PrefManager.Key.VIDEO_QUALITY, VideoQuality.AUTO.ordinal());` cuz default video quality is Auto.